.github/workflows/nameable.rb

```diff
@@ -0,0 +1,32 @@
+class Nameable
+    def correct_name
+      raise NotImplementedError, "Method 'correct_name' must be implemented in a subclass."
+    end
+  end
+  
+  class BaseDecorator < Nameable
+    def initialize(nameable)
+      @nameable = nameable
+    end
+  
+    def correct_name
+      @nameable.correct_name
+    end
+  end
+  
+
+  class CapitalizeDecorator < BaseDecorator
+    def correct_name
+      original_name = @nameable.correct_name
+      original_name.capitalize
+    end
+  end
+  
+  class TrimmerDecorator < BaseDecorator
+    def correct_name
+      original_name = @nameable.correct_name
+      trimmed_name = original_name[0..9] 
+      trimmed_name
+    end
+  end
+  
```

Comment:
**OPTIONAL** - Good job done so far, I suggest that you put each of your decorator classes in their own separate file. This would help make your repo more modular and easier to maintain. Please fix this 🙏

Reply:
Thanks for the feedback